### PR TITLE
test: update test_connect to use stop_app

### LINKED ISSUE #956: tests: update test/integration/connect/test_connect.py
Ensure that the `stop_app()` is called regardless of the success or failure of the tests.

### PR DESCRIPTION
**Pull request by dmyger**
Wrap call `start_app` & `stop_app` in try/finally blocks. Modified code was auto reformatted, also.

Closes #956

### CODE REVIEW

**Review by oleg-jukovec — APPROVED** [review]

**Review by themilchenko — COMMENTED** [review]
The error occurs in `test_connect`: https://github.com/tarantool/tt/actions/runs/11032062834/job/30640120785?pr=958#step:7:47

**Comment by oleg-jukovec** [comment]
Please, rebase the pull request.

**Review by oleg-jukovec — APPROVED** [review]

**Review by themilchenko — APPROVED** [review]
Thank you for the patch, looks good to me!

File: test/integration/connect/test_connect.py [hunk 1]
```diff
@@ -59,8 +59,8 @@ def start_app(tt_cmd, tmpdir_with_cfg, app_name, start_binary_port=False):
 
 
 def stop_app(tt_cmd, tmpdir, app_name):
-    stop_cmd = [tt_cmd, "stop", app_name]
-    stop_rc, stop_out = run_command_and_get_output(stop_cmd, cwd=tmpdir)
+    stop_cmd = [tt_cmd, "--no-prompt", "stop", app_name]
```
[the remark this answers is not in the document]
Comment (dmyger) [reply test/integration/connect/test_connect.py:62]: Done.

**Pull request merged**